transport-native-io_uring/src/main/c/netty_io_uring_native.c

```diff
@@ -296,8 +296,12 @@ static jobjectArray netty_io_uring_setup(JNIEnv *env, jclass clazz, jint entries
     return array;
 }
 
-static jint netty_create_file(JNIEnv *env, jclass class) {
-    return open("io-uring-test.txt", O_RDWR | O_TRUNC | O_CREAT, 0644);
+static jint netty_create_file(JNIEnv *env, jclass class, jstring filename) {
+    const char *file = (*env)->GetStringUTFChars(env, filename, 0);
+
+    int fd =  open(file, O_RDWR | O_TRUNC | O_CREAT, 0644);
```

Comment:
Since we're Linux specific (because `io_uring`) we can use `O_TMPFILE` and have the "file" refer to the parent directory. That way the file will never have a name, and will vanish once the fd is closed.

Reply:
good point ... let me do this

Reply:
It is a good idea but seems requires some checks to be sure it works
```
_TMPFILE requires support by the underlying filesystem;
              only a subset of Linux filesystems provide that support.
              In the initial implementation, support was provided in the
              ext2, ext3, ext4, UDF, Minix, and tmpfs filesystems.
              Support for other filesystems has subsequently been added
              as follows: XFS (Linux 3.15); Btrfs (Linux 3.16); F2FS
              (Linux 3.16); and ubifs (Linux 4.9)
```

Reply:
@franz1981 thanks a lot! Because of this I just did not change anything but added a explicit delete to the test.